ansible/library/minigraph_facts.py

```diff
@@ -325,7 +325,12 @@ def _parse_intf(intfname, ipprefix):
         intfs = []
         for ipintf in ipintfs.findall(str(QName(ns, "IPInterface"))):
             intfalias = ipintf.find(str(QName(ns, "AttachTo"))).text
-            intfname = port_alias_to_name_map.get(intfalias, intfalias)
+            if port_alias_to_name_map.has_key(intfalias):
+                intfname = port_alias_to_name_map[intfalias]
+            elif port_alias_asic_map.has_key(intfalias):
+                intfname = port_alias_asic_map[intfalias]
+            else:
+                intfname = intfalias
```

Comment:
Can you elaborate why this change is needed?  port_alias_asic_map is only populated in vs https://github.com/Azure/sonic-mgmt/blob/7c947a747b04c3de7c74b2779d8d91720e589a9a/ansible/module_utils/port_utils.py#L15. Is this change for vs?

Reply:
port_alias_asic_map is not just required for a vs platform, but all mulit-asic hosts. The reason that we see 'port_alias_asic_map' populated for only 'vs' platform in get_port_alias_to_name, because there is no support for any other multi-asic hwsku defined so far. Nokia is introducing a multi-asic linecard (Nokia-IXRE-36x400G) and for that linecard we also define the port_alias_asic_map - in PR https://github.com/Azure/sonic-mgmt/pull/4072